packages/data-point/lib/entity-types/entity-request/resolve.test.js

```diff
@@ -346,7 +346,7 @@ describe('resolve', () => {
     })
   })
 
-  test('inject locals value with string template', () => {
+  test('inject locals value with string template | contains object with {url: , and options: { baseUrl : } }', () => {
```

Comment:
This isn't injecting locals (see the other test above to see what that looks like).

This is more like `It should use options.baseURL to create a request URL`